src/modules/generateRandomDigit.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function generateRandomDigit() {
+  const digits = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const result = [];
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
Here you also have magic value 4 🙂
So you can place your constant in separate file and reuse